docs/content/concepts/sui-move-concepts/packages/automated-address-management.mdx

```diff
@@ -0,0 +1,85 @@
+
+Note that automated address management works for one package published to one or more chains. When you publish or upgrade, it uses the contents of the package's working directory. If a package is republished to a chain, addresses tracked for the previously published package will be overwritten for that chain.
+
+## Adopting automated address management for published packages
+
+Previously a `published-at` entry was **mandatory** in the `Move.toml` file, which sets the address of the latest published package. It is no longer required if this data is tracked in the `Move.lock` file. For an existing package, you'll need to add the necessary data to the `Move.lock` file so that it can be automatically tracked:
+
+1. Switch to your active environment for the chain that your package is published on
+
+```
+sui client --switch --env <YOUR-CHAIN-ENVIRONMENT>
+```
+
+2. Run the `manage-package` command with the data for your published package
+
+```
+sui move manage-package --environment "$(sui client active-env)" \
+                        --network-id "$(sui client chain-identifier)" \
+                        --original-id 'ORIGINAL-ADDRESS' \
+                        --latest-id 'LATEST-ADDRESS' \
+                        --version-number 'VERSION-NUMBER'
+```
+
+- `ORIGINAL-ADDRESS` should be the address your package was first published at. If you never upgraded your package, this the same address as your `published-at` address in the `Move.toml`
+- `LATEST-ADDRESS` should be the latest package address. If you never upgraded your package, it is the same as `ORIGINAL-ADDRESS`. If you upgraded your package, this is the same address as your current `published-at` address in the `Move.toml`.
+- `VERSION-NUMBER` is `1` if you never upgraded your package. Otherwise, it is a number greater than `1` depending on how many times you upgraded your package. In this case, look up the package at `LATEST-ADDRESS` to determine the version number.
+
+3. Delete the `published-at` line in your `Move.toml`. Your package is now tracked. You can repeat the above steps to track addresses for additional environments.
+
+## Package Upgrade Guidance
+
+- When [upgrading](./upgrade#upgrade-requirements), you will need to retrieve the `UpgradeCap` ID of your published package. Automated address management does not track your `UpgradeCap`.
+
+- When [upgrading](./upgrade#example), you will first need to set the `[addresses]` value for your package to `0x0` in the `Move.toml`, and restore its ID with the `ORIGINAL-ADDRESS` after upgrading.
```

Comment:
As mentioned, this should soon be unnecessary, just reflecting accuracy right now.